edb/schema/functions.py

```diff
@@ -970,6 +970,24 @@ def get_verbosename(
         sn = self.get_shortname(schema)
         return f"function '{sn}{params.as_str(schema)}'"
 
+    def get_dummy_body(self, schema: s_schema.Schema) -> expr.Expression:
+        rt = self.get_return_type(schema)
+
+        if rt.is_scalar():
+            # scalars and enums can be cast from a string
+            text = f'SELECT <{rt.get_displayname(schema)}>""'
```

Comment:
I'm not sure I understand how casting from empty string is correct here.

Reply:
Ah, this isn't supposed to be a valid expression in runtime. I suggest adding a docstring to this method explaining what's it for.